### PR TITLE
Add text/plain as a expected entry for content_type

### PR DESCRIPTION
**Pull request by kiranj2506**
Fixes zaproxy/zaproxy#7405

Signed-off-by: kiranj2506 <110659428+kiranj2506@users.noreply.github.com>

### CODE REVIEW

File: docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ var expectedTypes = [
 		"text/x-json",
 		"text/json",
 		"text/yaml"
```
Comment (psiinon) [inline docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js:27]: A comma is needed at the end of this line

**Comment by thc202** [comment]
The changelog should be updated.

**Comment by kiranj2506** [comment]
Thanks for the review @psiinon 
Added comma

**Comment by kingthorin** [comment]
Do you need us to do the [CHANGELOG](https://github.com/zaproxy/zaproxy/blob/main/docker/CHANGELOG.md) entry? I'm guessing based on the branch name that you're doing this via the web UI so changing another file might not be simple.

There was a similar change back on [2021-04-30](https://github.com/zaproxy/zaproxy/blob/main/docker/CHANGELOG.md#2021-04-30)

**Comment by kiranj2506** [comment]
@kingthorin thanks for the hint. I have added the changelog entry.

**Comment by kingthorin** [comment]
Thanks :+1:

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**